src/core/api/public_api.ts

```diff
@@ -51,6 +51,9 @@ import {
   isFullscreen,
   requestFullscreen,
 } from "../../compat";
+/* eslint-disable max-len */
```

Comment:
you can use `eslint-disable-next-line max-len` in the case there's only one line